Leet code interview150/leet code80

NTN77 opened the pull request.

hoangsang03 committed:
commit 1

NTN77 committed:
Update LC80_RemoveDuplicatesFromSortedArrayII.java

hoangsang03 committed:
add tasks for Nghia

there are 2 function that needs Nghia implements
1.countNumberRemainingValue
2.countNumberValueThatOccursMoreThan2Times

NTN77 committed:
LeetCode80

1-Method countNumberRemainingValue
2- Method countNumberValueThatOccursMoreThan2Times

hoangsang03 committed:
rename package

NTN77 committed:
UpDate

NTN77 committed:
UpDate_LeetCode1

hoangsang03 commented on src/test/java/Test_Top_Interview_Questions/TestLeetCode1_Nghia.java:
```diff
@@ -0,0 +1,27 @@
+    public void testTwoSum(){
+        int[] nums = {3,2,5,6};
+        int[] result = {1,2};
+        assertEquals(Arrays.toString(result),Arrays.toString(obj.twoSum(nums,7)));
```
https://junit.org/junit5/docs/5.0.0/api/org/junit/jupiter/api/Assertions.html
đọc thử tài liệu trên và dùng method **assertArrayEquals**(int[] expected, int[] actual)

hoangsang03 reviewed (commented):
Double check when testing whether two arrays are equal or not

NTN77 commented on src/test/java/Test_Top_Interview_Questions/TestLeetCode1_Nghia.java:
```diff
@@ -0,0 +1,27 @@
+    public void testTwoSum(){
+        int[] nums = {3,2,5,6};
+        int[] result = {1,2};
+        assertEquals(Arrays.toString(result),Arrays.toString(obj.twoSum(nums,7)));
```
Dạ để em học kĩ thêm

NTN77 committed:
update_TestLeetCode1

NTN77 committed:
Update_LeetCode_80

NTN77 committed:
Update_NTN

hoangsang03 committed:
resolve conflict

1. resolve conflict
2. create a new class for NHS

NTN77 committed:
Update

Rename Class

NTN77 committed:
update

Update code

hoangsang03 committed:
Update code for leetcode121 and LeetCode80

1. update the analyzation for leetcode121 solution
2. update code for leetcode121
3. update code for leetcode80
